src/BlockTypes/ProductGalleryLargeImageNextPrevious.php

```diff
@@ -40,83 +40,23 @@ protected function get_block_type_uses_context() {
 			case 'insideTheImage':
-				return array(
-					'class'       => 'inside-image',
-					'prev_button' => '<svg class="wc-block-product-gallery-large-image-next-previous-left--inside-image" xmlns="http://www.w3.org/2000/svg" width="49" height="48" viewBox="0 0 49 48" fill="none">
-							<g filter="url(#filter0_b_397_11356)">
-							<rect x="0.5" width="48" height="48" rx="5" fill="black" fill-opacity="0.5"/>
-							<path d="M28.1 12L30.5 14L21.3 24L30.5 34L28.1 36L17.3 24L28.1 12Z" fill="white"/>
-							</g>
-							<defs>
-							<filter id="filter0_b_397_11356" x="-9.5" y="-10" width="68" height="68" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
-							<feFlood flood-opacity="0" result="BackgroundImageFix"/>
-							<feGaussianBlur in="BackgroundImageFix" stdDeviation="5"/>
-							<feComposite in2="SourceAlpha" operator="in" result="effect1_backgroundBlur_397_11356"/>
-							<feBlend mode="normal" in="SourceGraphic" in2="effect1_backgroundBlur_397_11356" result="shape"/>
-							</filter>
-							</defs>
-							</svg>',
-					'next_button' => '<svg class="wc-block-product-gallery-large-image-next-previous-right--inside-image" xmlns="http://www.w3.org/2000/svg" width="49" height="48" viewBox="0 0 49 48" fill="none">
-							<g filter="url(#filter0_b_397_11354)">
-							<rect x="0.5" width="48" height="48" rx="5" fill="black" fill-opacity="0.5"/>
-							<path d="M21.7001 12L19.3 14L28.5 24L19.3 34L21.7001 36L32.5 24L21.7001 12Z" fill="white"/>
-							</g>
-							<defs>
-							<filter id="filter0_b_397_11354" x="-9.5" y="-10" width="68" height="68" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
-							<feFlood flood-opacity="0" result="BackgroundImageFix"/>
-							<feGaussianBlur in="BackgroundImageFix" stdDeviation="5"/>
-							<feComposite in2="SourceAlpha" operator="in" result="effect1_backgroundBlur_397_11354"/>
-							<feBlend mode="normal" in="SourceGraphic" in2="effect1_backgroundBlur_397_11354" result="shape"/>
-							</filter>
-							</defs>
-							</svg>',
-				);
+				return 'inside-image';
 			case 'outsideTheImage':
-				return array(
-					'class'       => 'outside-image',
-					'prev_button' => '<svg
-					width="22"
-					height="38"
-					viewBox="0 0 22 38"
-					fill="none"
-					xmlns="http://www.w3.org/2000/svg"
-					class=wc-block-product-gallery-large-image-next-previous-left--outside-image
-				>
-					<path
-						d="M17.7 0L21.5 3.16667L6.93334 19L21.5 34.8333L17.7 38L0.600002 19L17.7 0Z"
-						fill="black"
-					/>
-				</svg>',
-					'next_button' => '<svg
-					width="22"
-					height="38"
-					viewBox="0 0 22 38"
-					fill="none"
-					xmlns="http://www.w3.org/2000/svg"
-					class="wc-block-product-gallery-large-image-next-previous-right--outside-image"
-				>
-					<path
-						d="M4.56666 0L0.766663 3.16667L15.3333 19L0.766663 34.8333L4.56666 38L21.6667 19L4.56666 0Z"
-						fill="black"
-					/>
-				</svg>',
-				);
-
```

Comment:
The icons should be different based on the position of the arrow. This change introduces a regression:

| Mockup | Current Implementation |
|--------|--------|
|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/7f7828f3-bf1b-4638-a5a5-2e9bc7d96880)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/5e365b24-85dd-4322-94c1-a2ecf8017465)|
|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/eb9b2c22-d910-4fbc-ba44-23bee5adfc19)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/2dac2bea-8dd2-4af0-94ee-d2c0f8b1dae9)|

Reply:
Nice catch! It was solved in https://github.com/woocommerce/woocommerce-blocks/pull/10938/commits/f6cce8cfe2e0c8617085cd965e71a284d4dfac06